scripts/calculateRevenue/protocols/beefy/index.ts

```diff
@@ -113,7 +113,7 @@ export async function calculateVaultRevenue(
       BigInt(vaultTvl * 10 ** 18)
     const partialUsdContribution =
       Number(
-        (BigInt(tokenPriceUsd * 10 ** REVENUE_USD_PRECISION) *
+        (BigInt((tokenPriceUsd * 10 ** REVENUE_USD_PRECISION).toFixed(0)) *
```

Comment:
could also use parseUnits / formatUnits from viem for these conversions but those may need string handling

Reply:
thats a better solution. thanks!